### ISSUE TITLE
Platform Error After Updating To 2022.3.3

### ISSUE THREAD

**Issue by sbobergje** [syssi/xiaomi_fan#181]
```
Logger: homeassistant.config
Source: custom_components/xiaomi_miio_fan/fan.py:36
Integration: xiaomi_miio_fan (documentation, issues)
First occurred: 09:31:34 (1 occurrences)
Last logged: 09:31:34

Platform error: fan
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/config.py", line 877, in async_process_component_config
    platform = p_integration.get_platform(domain)
  File "/usr/src/homeassistant/homeassistant/loader.py", line 572, in get_platform
    cache[full_name] = self._import_platform(platform_name)
  File "/usr/src/homeassistant/homeassistant/loader.py", line 589, in _import_platform
    return importlib.import_module(f"{self.pkg_path}.{platform_name}")
  File "/usr/local/lib/python3.9/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1030, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1007, in _find_and_load
  File "<frozen importlib._bootstrap>", line 986, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 680, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/config/custom_components/xiaomi_miio_fan/fan.py", line 36, in <module>
    from miio import (  # pylint: disable=import-error
ImportError: cannot import name 'FanP9' from 'miio' (/usr/local/lib/python3.9/site-packages/miio/__init__.py)

```

**Comment by syssi** [comment]
This custom component isn't HA core 2022.3.3 yet. To fix some issue of the official `xiaomi_miio` component the underlyling library was updated and has undergone a few breaking changes. I will update this custom component asap!

**Comment by override80** [comment]
This worked in my case: https://github.com/syssi/xiaomi_fan/pull/183

**Comment by syssi** [comment]
Fixed by #183. I've prepared a new release. Please give it a try!
